content/pages/legal.md

```diff
@@ -1,172 +1,161 @@
 ---
 title: Legal Help
 slug: legal
-description: Guides for changing your name and gender markers in Alabama
+description: Guides for changing your name, gender markers, and more in Alabama.
 ---
 
-These guides are not written by lawyers, nor should they be interpreted as
-legal advice. They are based on the personal experiences of many members of
-the North Alabama transgender and non-binary community.
-
 [TOC]
 
+These guides are not written by lawyers, nor should they be interpreted as legal advice.
+They are based on the personal experiences of many members of the queer North Alabama community.
```

Comment:
This is probably a good time to drop North out of it, since we have a lot more people around AL
```suggestion
They are based on the personal experiences of many members of the Alabama trans community.
```